apps/emqx_bridge/src/emqx_bridge.erl

```diff
@@ -89,7 +89,8 @@
     T == pulsar_producer;
     T == oracle;
     T == iotdb;
-    T == kinesis_producer
+    T == kinesis_producer;
+    T == greptimedb_grpc_v1
```

Comment:
And the module should be version irrelevant, it needs to always be available no matter the driver's version.
The name could be `greptimedb`.